core/src/main/scala/com/databricks/labs/remorph/utils/Strings.scala

```diff
@@ -61,3 +62,16 @@ object Strings {
     }
   }
 }
+
+class Sed(rules: (String, String)*) {
+  private val compiledRules: Seq[(Regex, String)] = rules.map { case (regex, replace) =>
```

Comment:
If you decide to keep the current `(String, String)` API:
```suggestion
  private[this] val compiledRules: Seq[(Regex, String)] = rules.map { case (regex, replace) =>
```

Reply:
I'd have to change the transpiler as well, so I will change this to `private[this]`